test/demo/utils/globalSetup.ts

```diff
@@ -4,7 +4,7 @@ import * as path from 'path';
 
 import * as mkdirp from 'mkdirp';
 import fetch from 'node-fetch';
-import * as puppeteer from 'puppeteer';
+import puppeteer from 'puppeteer';
```

Comment:
Why this change?

Reply:
This meant I could get rid of the `"esModuleInterop": false` lines from `test/tsconfig.json`, which was a problem for the `deep-equal` import

Reply:
Well that's weird, but ok i guess